src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int maxNum = 101;
```

Comment:
```suggestion
    private static final int MAX_NUMBER = 101;
```
use the correct constant style